fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -474,6 +486,21 @@ private fun getMenuItems(
     return result
 }
 
+@VisibleForTesting
+internal fun submitTopSitesImpressionPing(topSite: TopSite.Provided, position: Int) {
```

Comment:
Since we aren't actually testing this, let's change this to `private fun`.

Reply:
ok changed to `private` we can change back when we add tests for the overall Composable `TopSite` section